### PR TITLE
Update expired status

### LINKED ISSUE #120: Set "expired" listings to `active: false`
When an listing has expired, we should set `active: false`. This essentially converts it into a draft listing until the listing calendar is updated so that the current date is within the availability dates.

### PR DESCRIPTION
**Pull request by syd-tan**
# Set "expired" listings to active: false 
Closes #120 

# Summary of change

Since this issue involves checking the listings collection daily and comparing the expiration date with the current date
as with an earlier issue #70, I refactored some of the calculations and just added another conditional block so that the function findExpiringListings() could meet the requirements for both issues. I also renamed reminder.helper.js to expiredListings.helper.js to better reflect these changes. 

# Testing/Verification

- Made sure the active status of listings whose expiration dates have passed were changed to false in the mongodb 
collection(changed them back to true just in case!)
- Verified that calculation of days past expiration were correct
- Also ensured that calculation still works for sending a reminder email a week before expiration

### CODE REVIEW

**Review by aidenszeto — APPROVED** [review]
Looks good to me - feel free to merge once @lullofthesea gets a chance to approve. Thanks!

**Comment by lullofthesea** [comment]
LGTM!  If possible, can you test with these values? `listing.available = [1625900400000, 1638259199999]`.  I'll check it on my own tomorrow then approve.  Good work!

**Comment by syd-tan** [comment]
Hey @lullofthesea, I think I saw a listing in the database with those availability times, and when I tested my function just now it seemed to work fine. Since the expiration date is November 30th, the listing active status remained true. Feel free to double check that! Thanks!

*I've also reverted any changes I made to the database just in case it'd interfere with other testing ^^

**Review by lullofthesea — APPROVED** [review]

**Pull request merged**
